content/version/2/0/code_of_conduct.nl.md

```diff
@@ -0,0 +1,93 @@
+
+We beloven om te handelen en te communiceren op een manier die bijdraagt aan een open, gastvrije, diverse, inclusieve en gezonde gemeenschap.
+
+## Onze standaarden
+
+Voorbeelden van gedrag dat bijdraagt ​​aan een positieve omgeving voor onze community omvatten:
+
+* Empathie en vriendelijkheid tonen tegenover andere mensen
+* Verschillende meningen, standpunten en ervaringen respecteren
+* Constructieve feedback geven en gracieus accepteren
+* Verantwoordelijkheid aanvaarden en excuses aanbieden aan degenen die door onze fouten zijn getroffen,
+  en leren van de ervaring
+* Focussen op wat het beste is voor de algemene gemeenschap, niet alleen voor ons als individuen
+
+Voorbeelden van onaanvaardbaar gedrag zijn:
+
+* Het gebruik van geseksualiseerde taal of beelden, en seksuele aandacht of voorschotten van welke aard dan ook
+* "Trollen", beledigende of denigrerende opmerkingen, en persoonlijke of politieke aanvallen
+* Openbare of privé-intimidatie
+* Publiceren van persoonlijke informatie van anderen, zoals een fysiek adres of e-mail adres, zonder hun uitdrukkelijke toestemming
+* Ander gedrag dat redelijkerwijs als ongepast zou kunnen worden beschouwd in een professionele setting
+
+## Handhaving
+
+Gemeenschapsleiders zijn verantwoordelijk voor het verduidelijken en handhaven van onze normen van acceptabel gedrag en zal passende en eerlijke corrigerende maatregelen nemen in reactie op elk gedrag dat zij ongepast, bedreigend, aanstootgevend, of schadelijk vinden.
+
+Gemeenschapsleiders hebben het recht en de verantwoordelijkheid om opmerkingen, commits, code, wiki-bewerkingen, issues en andere bijdragen die niet in overeenstemming met deze Gedragscode zijn te verwijderen, bewerken of afwijzen, en zal redenen voor moderatie meedelen waar nodig.
+
+## Toepassingsgebied
+
+Deze gedragscode is van toepassing binnen alle gemeenschappelijke ruimtes, en is ook van toepassing wanneer een individu officieel de gemeenschap in openbare ruimtes vertegenwoordigt.
+Voorbeelden van het vertegenwoordigen van onze gemeenschap zijn onder meer het gebruik van een officieel e-mailadres, posten via een officieel social media-account, of optreden als een aangestelde vertegenwoordiger bij een online of offline evenement.
+
+## Handhaving
+
+Beledigend, intimiderend of anderszins onaanvaardbaar gedrag kan gerapporteerd worden aan de gemeenschapsleiders die verantwoordelijk zijn voor handhaving via [CONTACTMETHODE INVOEGEN].
+Alle klachten zullen onmiddellijk en eerlijk worden beoordeeld en onderzocht.
+
+Alle gemeenschapsleiders zijn verplicht om de privacy en veiligheid van de verslaggever van een incident te respecteren.
+
+## Handhavingsrichtlijnen
+
+Gemeenschapsleiders zullen onderstaande "Community Impact Guidelines" volgen bij het bepalen van de consequenties van elke handeling die zij in strijd met de Gedragscode achten:
```

Comment:
I suggest to always use **Gedragscode** with a capital G. The EN translation also uses capitals.